config/initializers/omniauth.rb

```diff
@@ -0,0 +1,11 @@
+# config/initializers/omniauth.rb
+Rails.application.config.middleware.use OmniAuth::Builder do
+  if !Rails.env.production?
+    provider :developer,
+      :fields => [:uid, :name, :provider, :image],
+      :uid_field => :uid
+  else
+    provider :twitter, ENV["TWITTER_CONSUMER_KEY"], ENV["TWITTER_CONSUMER_SECRET"]
```

Comment:
good job using dot-env and secrets/keys to protect your API access! :key: :secret: :closed_lock_with_key: